site/app/libraries/Logger.php

```diff
@@ -175,77 +175,9 @@ private static function getTimestamp() {
     }
 
     /**
-     * This writes a one line message to the _access log file which we use to monitor what pages a user goes
-     * to (through a central point in the public/index.php file). This logs things in the fashion of:
-     *
-     * Timestamp | User ID | IP Address | Action | User Agent
-     *
-     * where action is defined broadly as the page they're accessing and any other relevant information
-     * (so gradeable id for when they're submitting).
-     */
-    public static function logAccess(string $user_id, string $token, string $action) {
```

Comment:
why are these functions being removed?

Reply:
If certain functions (e.g., logTAGrading, logQueueActivity) have overlapping purposes, the removals could be part of a refactor to simplify or streamline the logging system.

Security Enhancements:
Some removed lines might have been deemed risky in terms of exposing sensitive data (e.g., user agents, course details, etc.) to unauthorized access

The $token might be sensitive or unnecessary for logging purposes in this context. If the token is a security-related key or session identifier, logging it could pose a security risk by exposing it in log files.